cmake/Modules/FindComputeCpp.cmake

```diff
@@ -51,18 +50,13 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
     endif()
 endif()
 
-set(COMPUTECPP_64_BIT_DEFAULT ON)
-option(COMPUTECPP_64_BIT_CODE "Compile device code in 64 bit mode"
-        ${COMPUTECPP_64_BIT_DEFAULT})
-mark_as_advanced(COMPUTECPP_64_BIT_CODE)
-
-option(COMPUTECPP_DISABLE_GCC_DUAL_ABI "Compile with pre-5.1 ABI" OFF)
-mark_as_advanced(COMPUTECPP_DISABLE_GCC_DUAL_ABI)
-
 set(COMPUTECPP_USER_FLAGS "" CACHE STRING "User flags for compute++")
 mark_as_advanced(COMPUTECPP_USER_FLAGS)
 
-# Find OpenCL package
+set(COMPUTECPP_BITCODE "spir64" CACHE STRING
+  "Bitcode type to use as SYCL target in compute++")
+mark_as_advanced(COMPUTECPP_BITCODE)
```

Comment:
I understand that this could be very useful, but I can't think of that many situations where you'd want to build different targets in the same project with different bitcode formats (then there are questions like what happens when those targets are part of one object, but maybe it would just work?).

Reply:
It's more of a consistency thing with the rest of CMake. `COMPUTECPP_BITCODE` would set the default value of a target property with the same name, just like `CMAKE_CXX_STANDARD` sets the default value for the `CXX_STANDARD` target property.

I understand that this can be seen as a new feature and adding a target property shouldn't break backwards compatibility, so we can postpone this to a later PR if you like.

> then there are questions like what happens when those targets are part of one object, but maybe it would just work?

This can be handled by adding a potential `COMPUTECPP_BITCODE` target property to [`COMPATIBLE_INTERFACE_STRING`](https://cmake.org/cmake/help/v3.12/prop_tgt/COMPATIBLE_INTERFACE_STRING.html) (see also [here](https://cmake.org/cmake/help/v3.12/manual/cmake-buildsystem.7.html#compatible-interface-properties)).

Reply:
OK, on reflection, I think it's best that we make an issue for this, then add it as a later feature (: